src/server/engine_factory_v2.rs

```diff
@@ -80,6 +80,31 @@ impl TabletFactory<RocksEngine> for KvEngineFactoryV2 {
         None
     }
 
+    fn open_tablet_cache_latest(&self, id: u64) -> Option<RocksEngine> {
```

Comment:
The performance of this API is pretty bad----it requires O(N) to query all the keys to find the latest one.
Maybe it's better to have a map <region_id, latest_suffix> for a quick search.
Given its usage scenario is on the hot read path, the performance of this API is critical.

Reply:
Good catch and good suggestion.